src/main/java/edu/cornell/kfs/coa/batch/CuCoaBatchParameterConstants.java

```diff
@@ -0,0 +1,8 @@
+package edu.cornell.kfs.coa.batch;
+
+public class CuCoaBatchParameterConstants {
+    
+    public static final String CREATE_CLOSED_LABOR_ACCOUNTS_CSV_BY_PREVIOUS_DAYS_STEP = "createClosedLaborAccountsCsvByPreviousDaysStep";
```

Comment:
assuming the step name gets shortened, this will need updated as well

Reply:
Done.